src/cloudai/test_definitions/nemo_launcher.py

```diff
@@ -94,9 +94,8 @@ class NeMoLauncherCmdArgs(CmdArgs):
 
     repository_url: str = "https://github.com/NVIDIA/NeMo-Framework-Launcher.git"
     repository_commit_hash: str = "cf411a9ede3b466677df8ee672bcc6c396e71e1a"
-    docker_image_url: str = "nvcr.io/nvidian/nemofw-training:24.01.01"
+    docker_image_url: str = "nvcr.io/nvidia/nemo:24.05.01"
```

Comment:
Have you had a chance to test it to make sure we can run it through to the end? Typically, there is a working pair of docker image version and repository commit hash. Version 24.01.01 worked with commit cf411a9ede3b466677df8ee672bcc6c396e71e1a. I’m uncertain whether version 24.05.01 works with commit cf411a9ede3b466677df8ee672bcc6c396e71e1a.

Reply:
You can change this field in the test definition. This is the equivalent of the test template. Ideally the user should align this in their test definition.

Reply:
Yes, I am asking because the default value is important for user experience. I want to make sure the NeMo launcher works by default without requiring any additional changes. If you have not tested the pair (24.05.01 - cf411a9ede3b466677df8ee672bcc6c396e71e1a), let's revert this line.

Reply:
Fixed